### DISCUSSION TITLE
IP addresses of Vercel servers

### DISCUSSION THREAD

**Discussion by antoinebello** [vercel/next.js#54800] [Help] [answered]
### Summary

Our [site](https://thepopulationproject.org) is build on Next.js and hosted by Vercel. It makes lots of API calls to the back-end to build dynamic pages. As could be expected, our back-end receives a ton of unwanted calls that have nothing do with the site, yet use up our resources. 
Since the site is the only authorized user of the API, my idea would be to disallow all API calls other than those coming from Vercel's IP addresses. Is there such a list? I understand Vercel uses dynamic IP but still they must have a range of addresses.
Any help will be greatly appreciated :)

**Comment by leerob** [comment, marked as answer]
https://vercel.com/guides/how-to-allowlist-deployment-ip-address
